basiclti/basiclti-common/src/java/org/sakaiproject/util/foorm/Foorm.java

```diff
@@ -852,6 +859,21 @@ public void formOutputEnd(StringBuffer sb, String field, String label, Object lo
 		sb.append("</div>\n");
 	}
 
+	/**
+	 *
+	 * @param value
+	 * @param field
+	 * @param label
+	 * @param loader
+	 * @return
```

Comment:
complete this javadoc or remove

Reply:
This might be wrong but it matches the rest of Foorm for better or worse.